### ISSUE TITLE
Multiple callback errors when hovering over a projected map (no inverse_transform method)

### ISSUE THREAD

**Issue by niclasmattsson** [JuliaPlots/GeoMakie.jl#77]
Using the mouse to hover over the map generated in tests/runtests.jl produces the following error (Windows 10, GeoMakie 0.1.16). A new error is dumped for every mousemove event, so the REPL quickly gets filled with errors.

While waiting for a fix: is there an easy way to disable the hover callback for a given figure?

```
Error in callback:
MethodError: no method matching inverse_transform(::Makie.PointTrans{2, GeoMakie.PROJ{Projection, Projection}})
Closest candidates are:
  inverse_transform(::typeof(identity)) at C:\Stuff\Makie\src\layouting\transformation.jl:316
  inverse_transform(::typeof(log10)) at C:\Stuff\Makie\src\layouting\transformation.jl:317
  inverse_transform(::typeof(log)) at C:\Stuff\Makie\src\layouting\transformation.jl:318
  ...
Stacktrace:
  [1] process_interaction(r::Makie.MakieLayout.RectangleZoom, event::MouseEvent, ax::Axis)
    @ Makie.MakieLayout C:\Stuff\Makie\src\makielayout\interactions.jl:156
  [2] (::Makie.MakieLayout.var"#process_event#291"{Axis})(event::MouseEvent)
    @ Makie.MakieLayout C:\Stuff\Makie\src\makielayout\layoutables\axis.jl:399
  [3] #invokelatest#2
    @ .\essentials.jl:708 [inlined]
  [4] invokelatest
    @ .\essentials.jl:706 [inlined]
  [5] notify(observable::Makie.PriorityObservable{MouseEvent})
    @ Makie C:\Stuff\Makie\src\interaction\PriorityObservable.jl:72
  [6] setindex!
    @ C:\Stuff\Makie\src\interaction\PriorityObservable.jl:66 [inlined]
  [7] (::Makie.MakieLayout.var"#99#101"{Scene, Makie.MakieLayout.var"#is_mouse_over_relevant_area#96"{Scene, Tuple{}}, Base.RefValue{Float64}, Base.RefValue{Bool}, Base.RefValue{Bool}, Base.RefValue{Union{Nothing, Makie.Mouse.Button}}, Base.RefValue{Bool}, Base.RefValue{GeometryBasics.Point{2, Float32}}, Base.RefValue{GeometryBasics.Point{2, Float32}}, Base.RefValue{Makie.Mouse.Action}, Makie.PriorityObservable{MouseEvent}, Module})(mp::Tuple{Float64, Float64})
    @ Makie.MakieLayout C:\Stuff\Makie\src\makielayout\mousestatemachine.jl:212
  [8] #invokelatest#2
    @ .\essentials.jl:708 [inlined]
  [9] invokelatest
    @ .\essentials.jl:706 [inlined]
 [10] notify(observable::Makie.PriorityObservable{Tuple{Float64, Float64}})
    @ Makie C:\Stuff\Makie\src\interaction\PriorityObservable.jl:72
 [11] setindex!
    @ C:\Stuff\Makie\src\interaction\PriorityObservable.jl:66 [inlined]
 [12] (::GLMakie.var"#71#72"{Events, GLFW.Window})(#unused#::Nothing)
    @ GLMakie C:\Users\niclas\.julia\packages\GLMakie\ayWSm\src\events.jl:6
 [13] #invokelatest#2
    @ .\essentials.jl:708 [inlined]
 [14] invokelatest
    @ .\essentials.jl:706 [inlined]
 [15] notify
    @ C:\Users\niclas\.julia\packages\Observables\OFj0u\src\Observables.jl:88 [inlined]
 [16] setindex!(observable::Observable{Nothing}, val::Nothing)
    @ Observables C:\Users\niclas\.julia\packages\Observables\OFj0u\src\Observables.jl:248
 [17] fps_renderloop(screen::GLMakie.Screen, framerate::Float64)
    @ GLMakie C:\Users\niclas\.julia\packages\GLMakie\ayWSm\src\rendering.jl:22
 [18] renderloop(screen::GLMakie.Screen; framerate::Float64)
    @ GLMakie C:\Users\niclas\.julia\packages\GLMakie\ayWSm\src\rendering.jl:48
 [19] renderloop(screen::GLMakie.Screen)
    @ GLMakie C:\Users\niclas\.julia\packages\GLMakie\ayWSm\src\rendering.jl:41
 [20] (::GLMakie.var"#50#52"{GLMakie.Screen})()
    @ GLMakie .\task.jl:411
```

**Comment by abassign** [comment]
Hi, I have exactly the same problem with Julia 1.6.1 and Ubuntu 21.04 OS

**Comment by asinghvi17** [comment]
Should be fixed by #103 - I have implemented an inverse transform there.
